src/frontend/komponenter/Fagsak/Vedtak/OppsummeringVedtak.tsx

```diff
@@ -12,28 +13,50 @@ interface IVedtakProps {
 
 const OppsummeringVedtak: React.FunctionComponent<IVedtakProps> = ({ fagsak }) => {
     const [brev, setBrev] = React.useState('Genererer forhåndsvisning...');
+    const [errorMessage, setErrorMessage] = React.useState(undefined);
 
     React.useEffect(() => {
-        hentVedtaksbrev(fagsak.id)
+        hentVedtaksbrev(fagsak)
             .then((response: Ressurs<IVedtaksBrev>) => {
-                setBrev(response.data);
+                if (response.status === RessursStatus.SUKSESS) {
+                    setBrev(response.data);
+                    setErrorMessage(undefined);
+                } else {
+                    setErrorMessage(
+                        response.melding !== undefined
+                            ? response.melding
+                            : 'Kunne ikke generere forhåndsvisning.'
+                    );
+                }
             })
             .catch(error => {
-                setBrev('kunne ikke generere forhåndsvisning. Internal Error: ' + error);
+                const displayed = 'Internal Error: Kunne ikke generere forhåndsvisning.';
```

Comment:
Do we need to call `setBrev` here? If an error has occured we dont want to display the iframe anyway.

Reply:
No, I just forgot to remove it😁